e2e-tests/global-setup.js

```diff
@@ -83,6 +83,7 @@ export const test = baseTest.extend({
   sshUser: process.env.E2E_SSH_USER,
   targetAddress: process.env.E2E_TARGET_ADDRESS,
   targetPort: process.env.E2E_TARGET_PORT,
-  vaultAddr: process.env.E2E_VAULT_ADDR,
+  vaultAddr: process.env.E2E_VAULT_ADDR_PUBLIC,
+  vaultAddrPrivate: process.env.E2E_VAULT_ADDR_PRIVATE,
```

Comment:
When should we use which?

Reply:
Could we also update the usage of the vault address in the desktop test as well in this PR then?

Reply:
I don't believe any changes are needed in the desktop tests. They create credential stores without a worker filter, so the public address can be used.